### PR TITLE
GraphQL caching with Fastly

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds a section to "GraphQL Caching" that describes how to enable caching for Cloud/Fastly instances.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/caching.html

whatsnew
Added the [Caching with Fastly](https://devdocs.magento.com/guides/v2.3/graphql/caching.html) section in the "GraphQL Caching" topic in the _GraphQL Developer Guide_.

### CODE REVIEW

File: guides/v2.3/graphql/caching.md [hunk 1]
```diff
@@ -80,7 +80,17 @@ sub process_graphql_headers {
 
 [Configure Varnish and your web server]({{page.baseurl}}/config-guide/varnish/config-varnish-configure.html) further describes how to configure the `default.vcl` file.
 
+## Caching with Fastly
+
+{{site.data.var.ece}} uses Fastly for caching. To enable GraphQL caching on Fastly:
+
+1.  Upgrade the Fastly CDN Module for Magento 2.x to version 1.2.118 or higher.
+1.  Upload the VCL file to Fastly.
```
Comment (meker12) [inline guides/v2.3/graphql/caching.md]:
```suggestion
1.  Upload the updated VCL code to the Fastly servers.
```

File: guides/v2.3/graphql/caching.md [hunk 2]
```diff
@@ -80,7 +80,17 @@ sub process_graphql_headers {
 
 [Configure Varnish and your web server]({{page.baseurl}}/config-guide/varnish/config-varnish-configure.html) further describes how to configure the `default.vcl` file.
 
+## Caching with Fastly
+
+{{site.data.var.ece}} uses Fastly for caching. To enable GraphQL caching on Fastly:
+
+1.  Upgrade the Fastly CDN Module for Magento 2.x to version 1.2.118 or higher.
```
Comment (meker12) [inline guides/v2.3/graphql/caching.md]:
```suggestion
1. Upgrade the Fastly CDN Module for Magento 2.x to version 1.2.118 or higher.
```

File: guides/v2.3/graphql/caching.md [hunk 3]
```diff
@@ -80,7 +80,17 @@ sub process_graphql_headers {
...
+1.  Upload the updated VCL code to the Fastly servers.
```
Comment (meker12) [inline guides/v2.3/graphql/caching.md]:
```suggestion
1. Upload the updated VCL code to the Fastly servers.
```

**Comment by keharper** [comment]
running tests

**Pull request merged**